variables.tf

```diff
@@ -0,0 +1,120 @@
+variable "create_instance" {
+  type = bool
+  description = "Shether to create the alicloud_simple_application_server_instance"
+  default = false
+}
+
+variable "create_snapshot" {
+  type = bool
+  description = "Shether to create the alicloud_simple_application_server_snapshot"
+  default = false
+}
+
+variable "create_image" {
+  type = bool
+  description = "Shether to create the alicloud_simple_application_server_custom_image"
+  default = false
+}
+
+variable "instance_id" {
+  type = string
+  description = "the alicloud_simple_application_server_instance ID"
+  default = ""
+}
+
+variable "system_snapshot_id" {
+  type = string
+  description = "the System Snapshot ID"
+  default = ""
+}
+
+
+
+#alicloud_simple_application_server_instance
+variable "auto_renew" {
+  type = bool
+  description = "Specifies whether to enable auto-renewal. Unit: months. Valid values: true and false."
+  default = false
+}
+variable "auto_renew_period" {
+  type = number
+  description = "The auto renew period. Valid values: 1,3, 6, 12, 24, 36. NOTE: The attribute auto_renew is valid when the attribute is true."
+  default = null
+}
+variable "data_disk_size" {
+  type = number
+  description = "The size of the data disk. Unit: GB. Valid values: 0 to 16380."
+  default = null
+}
+variable "image_id" {
+  type = string
+  description = " The ID of the image. You can use the alicloud_simple_application_server_images to query the available images in the specified region. The value must be an integral multiple of 20."
```

Comment:
An existing simple application server image id. It will be ignored when create_image = true